sentinel-core/src/main/java/com/alibaba/csp/sentinel/config/ConfigLoadUtils.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.config;
+
+import com.alibaba.csp.sentinel.util.AppNameUtil;
+import com.alibaba.csp.sentinel.util.StringUtil;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.CopyOnWriteArraySet;
+
+/**
+ * @author lianglin
+ * @since 2019-05-31 15:19
+ */
+public class ConfigLoadUtils {
+
+    private static String DEFAULT_SENTINEL_CUSTOMIZE_FILE = "sentinel.properties";
+    private static String SENTINEL_CONFIG_FILE = "csp.sentinel.config.file";
+
+    private static final String DIR_NAME = "logs" + File.separator + "csp";
+    private static final String USER_HOME = "user.home";
+
+    private static Properties properties = new Properties();
+
+    static {
+        init();
+    }
+
+    public static Properties loadProperties() {
+        return properties;
+    }
+
+
+    private static void init() {
+
+        //Load from JVM -D user-specified file
+        String fileName = System.getProperty(SENTINEL_CONFIG_FILE);
+        if (StringUtil.isNotBlank(fileName)) {
+            if (fileName.startsWith("/")) {
```

Comment:
What if the `fileName` is a relative path?

Reply:
`                loadPropertiesFromResource(fileName);
`
deal with the relative path file

Reply:
may be the method name make confused, i will rename later